widgetastic_manageiq/expression_editor.py

```diff
@@ -186,8 +187,12 @@ def expression_text(self):
 
     def select_first_expression(self):
         """There is always at least one (???), so no checking of bounds."""
-        el = self.browser.elements(self.EXPRESSION_TEXT, parent=self._expressions_root)[0]
-        self.browser.click(el)
+        els = wait_for(
+            lambda: self.browser.elements(self.EXPRESSION_TEXT, parent=self._expressions_root),
+            fail_condition=[],
+            timeout=5,
+        )
+        self.browser.click(els[0][0])
```

Comment:
Need some in-line comments explaining these `0` indexes

Reply:
@mshriver thanks, Mike! Indeed, it doesn't look obvious.